sideboard/__init__.py

```diff
@@ -6,11 +6,7 @@
 import cherrypy
 
 from sideboard._version import __version__
-try:
-    import sideboard.server
-except:
-    from sideboard.lib import log
-    log.warning('Error importing server', exc_info=True)
+import sideboard.server
```

Comment:
This swallowing of exceptions at import time dates back to when Sideboard was still Dradis and we wanted to run in a non-server mode.  We handle that differently now, and this exception swallowing was just making it harder to debug Sideboard itself since I'd occasionally make a typo in ``server.py`` and then not notice right away.